app/src/main/java/ch/epfl/skysync/screens/crewpilot/FlightDetailsScreen.kt

```diff
@@ -35,7 +36,23 @@ fun CrewFlightDetailScreen(
           }
           is FinishedFlight -> {
             FinishedFlightDetailBottom(
-                reportClick = { navController.navigate(Route.REPORT + "/${flightId}") },
+                reportClick = {
+                  val reportDone =
```

Comment:
would be cleaner to have the whole logic in the ViewModel. Here you could add a function to the VM like for example:
`reportDoneFor(flightid, userId): Boolean` that can then simply be accessed through the VM

Reply:
👍